### PR TITLE
Create mods.tag

### PR DESCRIPTION
**Pull request by Greaper88**

### CODE REVIEW

**Comment by EasyClifton** [comment]
There seems to be a typo 
> play can
instead of
> player can

**Comment by Greaper88** [comment]
good catch, done

**Review by rtm516 — APPROVED** [review]

File: src/main/resources/tags/info/mods.tag [hunk 1]
```diff
@@ -0,0 +1,5 @@
+type: text
+
+---
+
+Geyser emulates a Vanilla Java game client. If the mods installed on the server allow a full vanilla client to connect, then a Geyser player can. (Conversely, if a player must install mods, a Geyser player can not join.)
```
Comment (Camotoy) [inline src/main/resources/tags/info/mods.tag:5]: Nitpick: `Vanilla Java game client` -> `vanilla Java game client`.

**Review by Camotoy — COMMENTED** [review]
Looks good to me otherwise.

File: src/main/resources/tags/info/mods.tag (hunk 1 above)
Comment (Greaper88) [reply src/main/resources/tags/info/mods.tag:5]: I personally have seen it "branded" like that a few times to distinguish it from non-vanilla server software.

Comment (Chew) [reply src/main/resources/tags/info/mods.tag:5]: Vanilla isn't a proper term, it's just a general term to clarify a non-modified client. It's more of an adjective to "Java" than "Vanilla Java" is one to "game client."

**Pull request merged**